dashboard/src/NoOperator.js

```diff
@@ -1,22 +1,23 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-import { Message } from 'semantic-ui-react';
+import { Container, Message, Modal, Segment } from 'semantic-ui-react';
 
 class NoOperator extends Component {
   render() {
     return (
-        <div className="App">
-          <header className="App-header">
-            <img src={logo} className="App-logo" alt="logo" />
-            <h1 className="App-title">Welcome to Kube-ArangoDB</h1>
-          </header>
-          <p className="App-intro">
-            There are no operators available yet.
-          </p>
-          {this.props.podInfoView}
-          {(this.props.error) ? <Message error content={this.props.error}/> : null}
-        </div>
+        <Container>
```

Comment:
This `NoOperator` component is exactly what I meant when I talked about presentational components btw. It doesn't have any state and it only contains display logic (i.e. "if there's an error message, put it here"). If you want to make it easier to swap out the styling later, you should try to extract all the actual presentation stuff into components like this one.

Minor nitpick: no point in using a class component here, you could just make it a function.